numba/jitclass/decorators.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, print_function
+from numba import types
+from .base import register_class_type, ClassBuilder
+from .immutable import ImmutableClassBuilder
+
+
+def jitclass(spec, immutable=False):
```

Comment:
By the way, we would probably like to pass other parameters here (at least `cache=True`, I think)...